x/superfluid/keeper/migrate.go

```diff
@@ -84,8 +88,6 @@ func (k Keeper) migrateSuperfluidBondedBalancerToConcentrated(ctx sdk.Context,
 		return 0, sdk.Int{}, sdk.Int{}, sdk.Dec{}, 0, 0, 0, err
```

Comment:
one of the thought process i had was since we only call the migration functions using `RouteLockedBalancerToConcentratedMigration` and `MigrateUnlockedPositionFromBalancerToConcentrated` does allow partial migration, i wanted to keep sharesToMigrate. For all the private migration functions like `migrateSuperfluidBondedBalancerToConcentrated` we can just default it to the entire lock coins which we do in `validateSharesToMigrateUnlockAndExitBalancerPool`

Reply:
Ah I see yeah, thank makes sense!